src/identity/interaction/email-password/storage/EmailPasswordStorageAdapter.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * A storage adapter needed for the email-password interaction
+ */
+export abstract class EmailPasswordStorageAdapter {
```

Comment:
```suggestion
export abstract class EmailPasswordStore {
```

Reply:
Changed to `EmailPasswordStore` and `ResourceStoreEmailPasswordStore`